Automattic/wp-calypso
# Signup: Create a new flow without the free plan, and with a skip button

This pull request fixes #3274 by creating a signup flow at /start/upgrade which doesn't contain the free plan:

<img width="872" alt="screen shot 2016-02-16 at 12 48 06" src="https://cloud.githubusercontent.com/assets/275961/13076649/8d3467ba-d4ab-11e5-8c7a-6f98fc74feb2.png">
 
Pinging @breezyskies and @mikeshelton1503 for a design review...

#### Testing instructions

1. Run `git checkout fix/3274-paid-plans-only` and start your server
2. Open the [`Upgrade signup flow` page](http://calypso.localhost:3000/start/upgrade)
3. Check that you can sign up for the free plan by using the "skip" button
3. Check that you can sign up for a paid plan by selecting a plan, and being sent to /checkout
3. Check that you can purchase a domain in signup

#### Additional notes
This also contains a change to the flow at /start/plan, adding the domain step, now that free trials are disabled there.

#### Reviews
